test

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+#
+# test - a pa test script
+
+export pa="$(pwd)/pa"
+
+export PA_DIR=test-stuff/passwords
+
+# clean up previous run
+# (the previous state is left around
+# intentionally, in case the dev
+# wants to poke around)
+# that's why we don't clean up on exit
+rm -rf test-stuff
+
+# pa welcomes you
+$pa | grep -q "a simple password manager" ||
+    printf "pa should print a welcome message\n"
+
+# generate pa dirs/identityfile/recipientfile
+$pa list
+
+# pa auto-generated files are correct
+test -s test-stuff/identities ||
```

Comment:
meant to be TDD in action lmao, i figure i might have time for the `PA_DIR` PR pretty soon, so maybe i figure this will remind and encourage me to do it haha.

Reply:
fair enough